src/plugins/condition/ConditionManager.js

```diff
@@ -108,7 +108,7 @@ export default class ConditionManager extends EventEmitter {
         if (conditionConfiguration) {
             conditionObj = {
                 ...conditionConfiguration,
-                id: uuid(),
+                id: conditionConfiguration.id || uuid(),
```

Comment:
This change is not needed as this is for the condition, not the criterion.

Reply:
done